### ISSUE TITLE
Compilation fails due to missing/incompatible dependency

### ISSUE THREAD

**Issue by nicmosc** [AndrewRedican/react-json-editor-ajrm#39]
1. What version of RJEA are you using (react-json-editor-ajrm version)?
`2.5.2`

2. What operating system and processor architecture are you using?
`Mac OS 10.13.6`

3. What did you do?
Tried to compile the code with webpack. This is the component usage I have:
```
import JSONInput from 'react-json-editor-ajrm';
import locale from 'react-json-editor-ajrm/locale/en';

<JSONInput
          id='1'
          locale={locale}
          height='100%'
      />
```

4. What did you expect to see?
The rendered page

5. What did you see instead?
Webpack throws this error
```
./node_modules/react-json-editor-ajrm/es/index.js
Module not found: Error: Can't resolve '@babel/runtime/helpers/builtin/es6/objectSpread' in '/Users/nick/Documents/Programming/repos/template-editor/node_modules/react-json-editor-ajrm/es'
```

**Comment by PatrickSachs** [comment]
Hey nicmosc!

Thanks for for reporting this. 👍

It seems like this is a duplicate of #37 which already has a fix waiting to be merged in #38. If that's not the case let me know, otherwise I'll close this issue so that we have one central ticket for this problem.

Until this issue is fixed you can manually install `@babel/runtime` at the **exact** version of `beta.55`.

**Comment by nicmosc** [comment]
@PatrickSachs thanks, looks like it's the same issue yeah. Manually installing that version works, i'll install the next version once the PR is merged! 👍🏻

**Comment by AndrewRedican** [comment]
Closing in favor of https://github.com/AndrewRedican/react-json-editor-ajrm/issues/37.
Fix suggested on https://github.com/AndrewRedican/react-json-editor-ajrm/pull/38 looks good.

This should be fixed shortly.

**Comment by AndrewRedican** [comment]
This has been fixed in the latest version `react-json-editor-ajrm@2.5.3`
